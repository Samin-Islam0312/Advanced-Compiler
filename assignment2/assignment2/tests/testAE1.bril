# ARGS: ae
@main {
.entry:
  a: int = const 1;
  b: int = const 2;
  t: int = add a b;          # add(a,b)
  u: int = add b a;          # canonicalizes to same expr as above
  print t;
  print u;
}

