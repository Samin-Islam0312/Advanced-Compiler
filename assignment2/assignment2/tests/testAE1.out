entry:
  in:  âˆ…
  out: add(a, b)
